### PR TITLE
chore(packages): remove arbiter from offline deployment packages

### PR DESCRIPTION
**Pull request by wuhuizuo**
Starts from v8.1.1

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

**Comment by wuhuizuo** [comment]
it should merge after release-8.3 is released.

File: packages/offline-packages.yaml.tmpl [hunk 1]
```diff
@@ -122,6 +122,7 @@ editions:
                   extract: true
                   extract_inner_path: reparo
               - name: arbiter # New in v6.0.0
+                if: {{ semver.CheckConstraint "< 8.1.1-0" .Release.version }}
```
Comment (Benjamin2037) [inline packages/offline-packages.yaml.tmpl:125]: I think we should remove only v8.1.0.

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: packages/offline-packages.yaml.tmpl (hunk 1 above)
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:125]:
```suggestion
                if: {{ semver.CheckConstraint "< 8.1.1-0 || >= 8.2.0-0" .Release.version }}
```

Comment (wuhuizuo) [reply packages/offline-packages.yaml.tmpl:125]: exclude the range: [8.1.1, 8.2.0)

File: packages/offline-packages.yaml.tmpl [hunk 2]
```diff
@@ -219,6 +220,7 @@ editions:
...
```
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:223]:
```suggestion
                if: {{ semver.CheckConstraint "< 8.1.1-0 || >= 8.2.0-0" .Release.version }}
```

File: packages/offline-packages.yaml.tmpl [hunk 3]
```diff
@@ -318,6 +320,7 @@ editions:
...
```
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:323]:
```suggestion
                if: {{ semver.CheckConstraint "< 8.1.1-0 || >= 8.2.0-0" .Release.version }}
```

File: packages/offline-packages.yaml.tmpl [hunk 4]
```diff
@@ -423,6 +426,7 @@ editions:
...
```
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:429]:
```suggestion
                if: {{ semver.CheckConstraint "< 8.1.1-0 || >= 8.2.0-0" .Release.version }}
```

File: packages/offline-packages.yaml.tmpl [hunk 5]
```diff
@@ -569,6 +573,7 @@ editions:
...
```
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:576]:
```suggestion
                if: {{ semver.CheckConstraint "< 8.1.1-0 || >= 8.2.0-0" .Release.version }}
```

File: packages/offline-packages.yaml.tmpl [hunk 6]
```diff
@@ -702,6 +707,7 @@ editions:
...
```
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:710]:
```suggestion
                if: {{ semver.CheckConstraint "< 8.1.1-0 || >= 8.2.0-0" .Release.version }}
```

File: packages/offline-packages.yaml.tmpl [hunk 7]
```diff
@@ -837,6 +843,7 @@ editions:
...
```
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:846]:
```suggestion
                if: {{ semver.CheckConstraint "< 8.1.1-0 || >= 8.2.0-0" .Release.version }}
```

File: packages/offline-packages.yaml.tmpl [hunk 8]
```diff
@@ -978,6 +985,7 @@ editions:
...
```
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:988]:
```suggestion
                if: {{ semver.CheckConstraint "< 8.1.1-0 || >= 8.2.0-0" .Release.version }}
```

File: packages/packages.yaml.tmpl [hunk 9]
```diff
@@ -998,6 +998,7 @@ components:
                 src:
                   path: bin/reparo
               - name: arbiter
+                if: {{ semver.CheckConstraint "< 8.1.1-0" .Release.version }}
```
Comment (wuhuizuo) [inline packages/packages.yaml.tmpl:1001]:
```suggestion
                if: {{ semver.CheckConstraint "< 8.1.1-0 || >= 8.2.0-0" .Release.version }}
```

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**